applications/openshift/master/file_groupowner_ovs_conf_db_not_s390x/rule.yml

```diff
@@ -1,7 +1,7 @@
 documentation_complete: true
 
 
-platform: ocp4-node and not_s390x_arch
+platform: ocp4-node and not_s390x_arch and not_aarch64_arch
```

Comment:
This can be reverted - included it accidentally. This rule appears to be working fine.

```
helpers.go:872: Result - Name: e2e-cis-node-worker-file-groupowner-ovs-conf-db - Status: PASS - Severity: medium
helpers.go:1060: Rule e2e-cis-node-worker-file-groupowner-ovs-conf-db matched expected result 
```

Reply:
Chasing this in a separate thread - https://issues.redhat.com/browse/OCPBUGS-53041